syntax = "proto3";

package descarteslabs.workflows;

enum ErrorCode {
  ERROR_NONE = 0;
  ERROR_UNKNOWN = 1;
  ERROR_INVALID = 2;
  ERROR_DEADLINE = 3;
  ERROR_OOM = 9;
  ERROR_INTERRUPT = 15;
  ERROR_TERMINATED = 16;
  ERROR_AUTH = 17;
  ERROR_TYPING = 18;
  ERROR_SERIALIZATION = 19;
  ERROR_DESTINATION = 20;
  ERROR_TRANSIENT = 21;
}
